docker/entrypoint_foxx.sh

```diff
@@ -33,17 +33,32 @@ then
   # Check to see if foxx has previously been installed
   "${PROJECT_ROOT}/scripts/generate_datafed.sh"
 
-  "${DATAFED_DEPENDENCIES_INSTALL_PATH}/bin/cmake" -S. -B build						\
-    -DBUILD_REPO_SERVER=False		\
-    -DBUILD_COMMON=False        \
-    -DBUILD_AUTHZ=False					\
-    -DBUILD_CORE_SERVER=False		\
-    -DBUILD_WEB_SERVER=False		\
-    -DBUILD_DOCS=False					\
-    -DBUILD_PYTHON_CLIENT=False	\
-    -DBUILD_FOXX=True           \
-    -DINSTALL_FOXX=True
-
+  # Should only run this if you are ok with making changes to the database
+  if [ "$ENABLE_FOXX_TESTS" == "TRUE" ]
+  then
+    "${DATAFED_DEPENDENCIES_INSTALL_PATH}/bin/cmake" -S. -B build \
+      -DBUILD_REPO_SERVER=False   \
+      -DBUILD_COMMON=False        \
+      -DBUILD_AUTHZ=False         \
+      -DBUILD_CORE_SERVER=False   \
+      -DBUILD_WEB_SERVER=False    \
+      -DBUILD_DOCS=False          \
+      -DBUILD_PYTHON_CLIENT=False \
+      -DBUILD_FOXX=True           \
+      -DINSTALL_FOXX=True         \
+      -DENABLE_FOXX_TESTS=True
+  else
+    "${DATAFED_DEPENDENCIES_INSTALL_PATH}/bin/cmake" -S. -B build \
+      -DBUILD_REPO_SERVER=False   \
+      -DBUILD_COMMON=False        \
+      -DBUILD_AUTHZ=False         \
+      -DBUILD_CORE_SERVER=False   \
+      -DBUILD_WEB_SERVER=False    \
+      -DBUILD_DOCS=False          \
+      -DBUILD_PYTHON_CLIENT=False \
+      -DBUILD_FOXX=True           \
+      -DINSTALL_FOXX=True
+  fi
```

Comment:
if `-DENABLE_FOXX_TESTS=True` is the only difference between `ENABLE_FOXX_TESTS` being true and false, can you do something like this with an inplace variable update?


```shell
cmake_options="-S. -B build \
  -DBUILD_REPO_SERVER=False   \
  -DBUILD_COMMON=False        \
  -DBUILD_AUTHZ=False         \
  -DBUILD_CORE_SERVER=False   \
  -DBUILD_WEB_SERVER=False    \
  -DBUILD_DOCS=False          \
  -DBUILD_PYTHON_CLIENT=False \
  -DBUILD_FOXX=True           \
  -DINSTALL_FOXX=True"

if [ "$ENABLE_FOXX_TESTS" == "TRUE" ]; then
  cmake_options="$cmake_options -DENABLE_FOXX_TESTS=True"
fi

"${DATAFED_DEPENDENCIES_INSTALL_PATH}/bin/cmake" $cmake_options
```

Reply:
How would you feel about me putting this in a follow up PR?

Reply:
https://github.com/ORNL/DataFed/issues/1063